ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/cvx.j2

```diff
@@ -0,0 +1,33 @@
+{# CVX #}
+{% if cvx is arista.avd.defined %}
+
+# CVX
+{%     if cvx.shutdown is arista.avd.defined(true) %}
+
+CVX is disabled
+{%     elif cvx.shutdown is arista.avd.defined(false) %}
+
+CVX is enabled
+{%         if cvx.services is arista.avd.defined %}
+
+## CVX services
+
+| Service | Enabled | Settings |
+| ------- | ------- | -------- |
+{%             if cvx.services.mcs is arista.avd.defined %}
+{%                 set enabled = not cvx.services.mcs.shutdown | arista.avd.default('-') %}
+{%                 set settings = [] %}
+{%                 if cvx.services.mcs.redis.password is arista.avd.defined %}
+{%                     do settings.append("Redis Password Set") %}
+{%                 endif %}
```

Comment:
To have specific columns, we need a table per cvx service, since they all differ in their supported settings. Would be ok for me. But then I think we should also all the enabled/disabled (based on `not cvx.services.mcs.shutdown`) to that table instead of having it above.